src/AbstractSQLCompiler.ts

```diff
@@ -103,15 +103,18 @@ export type CoalesceNode = [
 	UnknownTypeNodes,
 	...UnknownTypeNodes[],
 ];
+export type ToJSONNode = ['ToJSON', UnknownTypeNodes];
 export type UnknownTypeNodes =
 	| FieldNode
 	| ReferencedFieldNode
 	| BindNode
 	| CastNode
 	| CoalesceNode
+	| ToJSONNode
 	| UnknownNode;
 
 export type TextNode = ['Text', string];
+export type JSONNode = ['JSON', string];
```

Comment:
Not sure how to define `JSONNode`, from what I understand Postgres treats `JSON` columns roughly the same as `TEXT`, just with basic JSON validation.

Reply:
I think we can treat it the same here and rely on postgres complaining